app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/SprintsRepositoryTest.kt

```diff
@@ -0,0 +1,192 @@
+package io.eugenethedev.taigamobile.repositories
+
+import io.eugenethedev.taigamobile.data.repositories.SprintsRepository
+import io.eugenethedev.taigamobile.domain.repositories.ISprintsRepository
+import io.eugenethedev.taigamobile.testdata.TestData
+import kotlinx.coroutines.runBlocking
+import org.junit.Test
+import java.time.LocalDate
+import kotlin.test.BeforeTest
+import kotlin.test.assertEquals
+
+class SprintsRepositoryTest: BaseRepositoryTest() {
+    lateinit var sprintsRepository : ISprintsRepository
+
+    @BeforeTest
+    fun setupSprintsRepositoryTest() {
+        sprintsRepository = SprintsRepository(mockTaigaApi, mockSession)
+    }
+
+    @Test
+    fun `test get user stories`() = runBlocking {
+        val userStories1 = sprintsRepository.getSprintUserStories(1)
+        val userStories2 = sprintsRepository.getSprintUserStories(2)
+
+        val testUserStories = TestData.projects[0].userstories.filter { it.sprint != null }
+
+        assertEquals(
+            expected = 1,
```

Comment:
And what if we will add more user stories to sprint in TestData in future? Although it is good that you are checking list size before the next assert (which implies that list has exactly one element), the practice is actually bad. You should not hardcode test data here in test, because we have single source of truth about all test data - TestData object. So I would recommend the same approach I used [here](https://github.com/EugeneTheDev/TaigaMobile/blob/master/app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/SearchRepositoryTest.kt#L23). Leave only two asserts (one for each sprint) which check that two name lists (ours and actual from API) are the same. Or you can check not only names, but also dates, for instance. You get the idea